plugins/org.eclipse.elk.core/src/org/eclipse/elk/core/Core.melk

```diff
@@ -285,6 +285,18 @@ advanced option interactiveLayout: boolean {
     targets parents
 }
 
+advanced option omitNodeMicroLayout: boolean {
+    label "Omit Node Micro Layout"
+    description
+        "Node micro layout comprises the computation of node dimensions (if requested), the placement of ports 
```

Comment:
"if requested" refers to node size constraints, right? I think it would be good to reference them here.